app/views/layouts/application.html.erb

```diff
@@ -174,8 +174,5 @@
         </div>
       </div>
     </footer>
-
-    <%= javascript_include_tag 'js/main.js' %>
```

Comment:
Did this previously not exist? 🤔

Reply:
yeah, this was asking for something that does not exist

Reply:
actually, double checking, this may be USWDS JS. Let me make sure I haven't broken everything

Reply:
> `identity-style-guide` is in the asset path and this is pulling from there.

Ah, tricky!